src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -115,19 +124,29 @@ private static List<(string uid, long start, long end)> GetXrefSpecPositions(Rea
                         {
                             stack.Push((Encoding.UTF8.GetString(reader.ValueSpan), top.start));
                         }
+                        if (propertyNames.Length > 0 && stack.Count == 1)
+                        {
+                            foreach (var pname in propertyNames)
+                            {
+                                if (reader.ValueTextEquals(Encoding.UTF8.GetBytes(pname)) && reader.Read() && reader.TokenType == JsonTokenType.String)
+                                {
```

Comment:
When the property name is `repositoryUrl` and the reader type is not a string, throw `Errors.UnexpectedType` to stop the load process. Because if the xrefmap looks like `{ repositoryUrl: { }, ... }`, this method may throw later because this line has eaten the `StartObject` condition and corrupt the `stack`.

Reply:
OK.